src/nupic/frameworks/opf/exp_description_api.py

```diff
@@ -60,35 +59,36 @@
 
 
 class DescriptionIface(object):
-  """ This is the base interface class for description API classes which provide
+  """
+  This is the base interface class for description API classes which provide
   OPF configuration parameters.
 
   This mechanism abstracts description API from the specific description objects
-  created by the individiual users.
+  created by the individual users.
 
   TODO: logging interface?
+
+  :param modelConfig: (dict)
+      Holds user-defined settings for model creation.  See OPF
+      `here <description-template.html>`_ for config dict documentation.
+
+  :param control: (dict)
+      How the model is to be run. The schema of this dictionary depends on the
+      'environment' parameter, which specifies the context in which the model is
+      being run.
   """
   __metaclass__ = ABCMeta
 
 
   @abstractmethod
   def __init__(self, modelConfig, control):
-    """
-        modelConfig:
-            a dictionary object which holds user-defined settings for model
-            creation.  See OPF descriptionTemplate.tpl for config dict
-            documentation
-
-        control:
-            A dictionary describing how the model is to be run. The schema of
-            this dictionary depends on the 'environment' parameter, which
-            specifies the context in which the model is being run.
-    """
+    pass
```

Comment:
Is there something in the tooling that required shifting the docstring from `__init__` to the class declaration?  It seems odd (to me) to document constructor arguments there.